src/widgets/sort-by/sort-by.js

```diff
@@ -12,10 +11,12 @@ const renderer = ({ containerNode, cssClasses }) => (
   { currentRefinement, options, refine },
   isFirstRendering
 ) => {
-  if (isFirstRendering) return;
+  if (isFirstRendering) {
+    return;
+  }
 
   render(
-    <div className={cx(cssClasses.root)}>
+    <div className={cssClasses.root}>
```

Comment:
not sure I understand the reasoning of these changes?

Reply:
Inside the widget (below) we precompute the class we don't have to run them through the function again. See https://github.com/algolia/instantsearch.js/pull/3169/files#diff-f4506bac0a67ca481cb05bddf147f9c7R96